src/test/java/com/easypost/ReportTest.java

```diff
@@ -64,13 +66,36 @@ public static void setup() throws EasyPostException {
         globalTrackerReport = Report.create(trackerParams);
     }
 
+    @Test
+    public void testCreateReportWithAdditionalColumns() throws EasyPostException {
```

Comment:
This is fine, please also add a test that only passes the `columns` param.

Reply:
added